website/public/locales/th/index.json

```diff
@@ -1,23 +1,23 @@
 {
   "blurb": "เราเชื่อว่าเราสร้างการปฏิวัติได้",
   "blurb1": "ในลักษณะเดียวกับที่ Stable Diffusion ช่วยให้โลกสร้างงานศิลปะและรูปภาพในรูปแบบใหม่ เราต้องการปรับปรุงโลกด้วยการเสนอ Conversational AI ที่น่าทึ่ง",
-  "description": "Conversational AI สำหรับทุกคน โครงการโอเพ่นซอร์สเพื่อสร้างการแชทที่เปิดใช้งาน GPT LLM ดำเนินการโดย LAION และผู้ร่วมให้ข้อมูลทั่วโลก",
+  "description": "AI เชิงสนทนา (Conversational AI) สำหรับทุกคน โปรเจกต์โอเพ่นซอร์สเพื่อสร้างการแชทที่เปิดใช้งาน GPT LLM ดำเนินการโดย LAION และผู้มีส่วนร่วมทั่วโลก",
   "faq_items": {
-    "q0": "ในระยะยาวโครงการนี้จะเป็นอย่างไร ?",
+    "q0": "โปรเจกต์ไปได้ไกลถึงไหนแล้ว?",
     "a0": "เรากำลังอยู่ในช่วงเริ่มต้นของการพัฒนา โดยทำงานจากงานวิจัยที่เป็นที่ยอมรับในการใช้ RLHF กับโมเดลภาษาขนาดใหญ่",
     "q1": "ใครอยู่เบื้องหลัง Open Assistant?",
-    "a1": "Open Assistant เป็นโครงการที่ดำเนินงานโดย LAION และบุคคลทั่วโลกที่สนใจนำเทคโนโลยีนี้ไปให้ทุกคน",
+    "a1": "Open Assistant เป็นโปรเจกต์ที่ดำเนินงานโดย LAION และบุคคลทั่วโลกที่สนใจนำเทคโนโลยีนี้ไปให้ทุกคน",
     "q2": "Open Assistant ใช้ใบอนุญาตอะไร?",
-    "a2": "ชุดคำสั่งและแบบจำลองอยู่ภายใต้ใบอนุญาต Apache 2.0",
-    "q3": "ข้อมูลที่ใช้ในการทดสอบแบบจำลองจะถูกเปิดเผยด้วยใช่ไหม?",
+    "a2": "ตัวซอสโค๊ดและโมเดลอยู่ภายใต้ใบอนุญาต Apache 2.0",
```

Comment:
ผมมองว่าข้อความเดิมเหมาะสมอยู่แล้วนะครับ

I think this old text is better.

Reply:
อันนี้ผมว่าจะเข้าง่ายกว่านะ ส่วนตัวยังไม่เคยได้ยินคนใช้คำว่า "ชุดคำสั่ง" และ "แบบจำลอง"

Reply:
ในวงวิชาการเวลาเขียนเอกสารวิชาการใช้งานกันนะครับ ส่วนตัวผมคิดว่า ควรยึดภาษาเขียนเป็นหลักมากกว่าครับ

Reply:
อันนี้ไม่ใช่วิชาการครับ เราควรจะแปลให้คนอ่านเข้าใจง่าย